.github/ISSUE_TEMPLATE/hackathon-project-template.md

```diff
@@ -10,41 +10,58 @@ assignees: '@ohbm/hack2022-monitors'
+ 2. *Submit the issue*
+ 3. *Check items in the checklist*
+ 4. *Once you are done (at least all 'required' items must be provided), please delete the "Guidelines" section add a comment saying 'hi @ohbm/project-monitors: My project is ready!'*
 
 Thank you!
 
-*After step 1 (issue submitted), we will assign a 'project monitor' to follow your submission. If at any time you need help or anything is unclear, please add a comment and ping your project monitor. Our team is here to help!*
+*After submission, we will assign a 'project monitor' to follow your submission. If at any time you need help or anything is unclear, please add a comment and ping your project monitor. Our team is here to help!*
 
 
 ----------------------------
 ## Project info
-<!-- *Please fill this in first and then submit the issue* -->
+<!-- *Please fill this in first and then submit the issue* 
+See [here](https://github.com/ohbm/hackathon2022/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#link-to-project)
+-->
 
 **Title**:
 <!--Name of your awesome project. Please also update the title of the issue to be the title of your project-->
 
-**Project lead**:
-<!--Your name and GitHub login, possibly more than 1 lead-->
+**Project leaders with Mattermost handle and GitHub login**:
+<!--Please provide more than one leader if possible and include the [Mattermost handle](https://mattermost.brainhack.org/) (i.e. your username). 
+If you do not have an account, please [sign up here](https://mattermost.brainhack.org/signup_email).
+Include the GitHub login if you are planning to use.
+-->
 
-**[Timezone](https://github.com/ohbm/hackathon2022/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#timezone)**:
-<!--UTC offset of your timezone (cf. https://www.timeanddate.com/time/map/ for example).-->
+**[In-person/online]**:
+<!--Provide information if leaders are planning to be in-person and/or online. 
+If online, please provide your timezone (cf. https://www.timeanddate.com/time/map/ for example).
+-->
 
 **Description**:
 <!--Describe the main idea and context of your project in a few sentences.-->
 
 **Link to project**:
 
-**Mattermost handle**:
-
 **Goals for the OHBM Brainhack**
+<!-- Describe what you want to achieve during this brainhack, 
+see [here](https://github.com/ohbm/hackathon2022/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#goals)
+-->
+
+**Skills**:
+<!--Provide a list of the skills, you can specify required and recommended, 
+see [examples](https://github.com/ohbm/hackathon2022/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-skills)
+-->
+
+**List of recommended tutorials**
+<!--TODO/trainhack-->
 
 **Good first issues**:
+<!--If you would like to welcome new people, please provide a list of good-first-issues-->
 
-**Skills**:
 
 **Chat channel**:
 <!-- If you are creating a channel on the [brainhack mattermost](https://mattermost.brainhack.org/) try to create a
```

Comment:
be clear that this will be created for them, but they can suggest the name

Reply:
We can add a link to the [traintrack material](https://ohbm.github.io/hackathon2022/traintrack/) and suggest the project leader to have a look.